pretyping/structures.mli

```diff
@@ -53,6 +53,10 @@ val projection_nparams : Names.Constant.t -> int
 
 val is_projection : Names.Constant.t -> bool
 
+val projection_number : Environ.env -> Names.Constant.t -> int
+(** [projection_number env p] returns the position of the projection p in
+    the structure it corresponds to. *)
```

Comment:
It should not be necessary to write it down, but in this context I would add that we count from 0.